src/state/routing/utils.ts

```diff
@@ -90,6 +96,17 @@ export function transformQuoteToTradeResult(args: GetQuoteArgs, quoteResult: Quo
           inputAmount,
           outputAmount,
         })) ?? [],
+    mixedRoutes:
+      routes
+        ?.filter(
+          (r): r is typeof routes[0] & { mixedRoute: NonNullable<typeof routes[0]['mixedRoute']> } =>
+            r.mixedRoute !== null
+        )
+        .map(({ mixedRoute, inputAmount, outputAmount }) => ({
```

Comment:
what is the point of this `.map` operation? it looks like you're not actually changing the value before returning it?

Reply:
i think it's to remove the extraneous fields from each `route` object to match the Typescript types